packages/forms/resources/views/components/tabs.blade.php

```diff
@@ -57,6 +57,10 @@
                 class="shrink-0 p-3 text-sm font-medium"
                 x-bind:class="{ 'bg-white @if (config('forms.dark_mode')) dark:bg-gray-800 @endif': tab === '{{ $tab->getId() }}' }"
             >
+                @if ($tab->hasIcon())
+                    <x-dynamic-component :component="$tab->getIcon()" class="h-5 w-5 mr-1 text-gray-500 @if (config('forms.dark_mode')) dark:text-gray-400 @endif inline"/>
```

Comment:
I feel like the icon should be a bit smaller, looks a bit too big next to the text.

Reply:
I agree, also the solid Heroicons should be used instead of outline in this context IMO, which could also affect sizing

Reply:
Exactly what @danharrin says. The sizing classes are in fact correct, but the solid icons should be used.